spec/models/dataset_file_spec.rb

```diff
@@ -4,10 +4,21 @@
 
 RSpec.describe DatasetFile do
   describe "#download_url" do
-    it "builds the proper URL" do
+    it "builds the proper URL for a DataSpace record" do
       hash = { name: "big file.zip", handle: "123/dsp456", sequence: 1 }
       file = described_class.from_hash(hash, "dataspace")
-      expect(file.download_url).to eq "https://dataspace-dev.princeton.edu/bitstream/123/dsp456/1/big file.zip"
+      expect(file.download_url).to eq "https://dataspace-dev.princeton.edu/bitstream/123/dsp456/1"
+    end
+
+    it "builds the proper URL for a PDC Describe record" do
+      hash = {
+        name: "10.34770/qyrs-vg25/50/file_name.txt",
+        full_name: "file_name.txt",
+        size: 455_511,
+        url: "https://g-ef94ef.f0ad1.36fe.data.globus.org/10.34770/qyrs-vg25/50/file_name.txt"
+      }
+      file = described_class.from_hash(hash, "pdc_describe")
+      expect(file.download_url).to eq hash[:url]
```

Comment:
This test is new and has nothing to do with the PR itself, but I wanted to make sure we test PDC Describe records too.